src/chat_obs/components/CameraView.tsx

```diff
@@ -98,7 +105,7 @@ export const CameraView = forwardRef<CameraViewRef, CameraViewProps>(
         id: id,
         _id: id,
         type: media.type,
-        path: media.path,
+        path: getAbsoluteFilePath(media.path),
```

Comment:
sometime it is not `media.path` but `media.sourceURL`

Reply:
`PhotoFile` and `VideoFile` type from `react-native-vision-camera` doesn't have `sourceURL` property

Reply:
Okay then, if you can assure that